src/Backend/Core/Layout/Sass/_bootstrap-variables.scss

```diff
@@ -934,10 +935,10 @@ $toast-header-border-color:         rgba(0, 0, 0, .05);
 
 // Badges
 
-$badge-font-size:                   75%;
+$badge-font-size:                   $font-size-base * 0.75;
```

Comment:
I would leave this in percentage because a badge is a span-tag, and is 75% of the font-size of the tag that it is in. P.ex. a badge in a title is larger than a badge in a p-tag, which looks better than badges that are always `$font-size-base * 0.75`

Reply:
valid point